### ISSUE TITLE
Show build configurations where builds fail to start

### ISSUE THREAD

**Issue by janv8000** [amoerie/teamcity-theatre#39]
Possible causes of builds failing to start: VCS checkout, ...

![image](https://user-images.githubusercontent.com/1412705/48054973-c9bd8500-e1ae-11e8-94e6-063b14c468f3.png)

Right now only the started builds that are not succesful cause the monitor to turn red...

**Comment by amoerie** [comment]
Good point, but that sounds like a configuration problem in your TeamCity instance. TCT would have to make separate API calls just to fetch this information, because it doesn't fall under the umbrella of builds.

I'm on the fence whether this sort of thing belongs in TCT.

**Comment by amoerie** [comment]
After some consideration I think this is very relevant. I'll try to include it in the next release.
